extensions/2.0/Khronos/KHR_texture_video/KHR_texture_video.md

```diff
@@ -0,0 +1,155 @@
+# KHR_texture_video
+
+## Contributors
+
+* Dwight Rodgers, Adobe
+* Gary Hsu, Microsoft
+* Emmett Lalish, Google
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension allows glTF models to use videos as textures. A client that does not implement this extension can ignore the provided video and continue to rely on the PNG and JPG textures available in the base specification. Defining a fallback texture is optional. 
+
+Video textures support use cases such as water or other textures that continuously repeat, coordination of video texture playback by animation (in conjunction with `KHR_animation_pointer`), or control of video texture playback by interactivity (in conjunction with `KHR_interactivity`).
```

Comment:
Where is `KHR_interactivity` defined? It's not a part of this repository.

For interactivity, note that [OMI's physics extensions](https://github.com/KhronosGroup/glTF/pull/2258) already define an interactible trigger. You can specify a trigger shape that a user can point at in the world and interact with. What actually gets triggered is defined by other extensions, so this video player extension could define that a given OMI trigger plays the video. I would highly recommend that we move forward with OMI physics so that we can use these extensions together.

Reply:
KHR_interactivity is an ongoing work by the 3d format's group's DTSG on behaviors and interactivity. It is still an early work in progress, but a partial draft should be available on a branch soon.